ui/widgets/nuxeo-selectivity.js

```diff
@@ -3464,8 +3464,12 @@ typedArrayTags[weakMapTag] = false;
         _updateInputWidth() {
           const inputContent =
             this.input.value || (!this._data.length && this.options.placeholder) || '';
-          this.input.setAttribute('size', inputContent.length + 2);
-
+            const placeholderLength = typeof this.options.placeholder !== 'undefined' && this.options.placeholder.length;
```

Comment:
I don't think we benefit here from using "typeof ... !== 'undefined'" here, since `this` is always defined, and it is harder to read than just comparing to `undefined` or `null`.  We can compare against both by using `!=` as follows: 
```suggestion
            const placeholderLength = this.options.placeholder != null && this.options.placeholder.length;
```

Reply:
thankyou @Gabez0r ,made the changes.